provisioner/terraform/resources_test.go

```diff
@@ -636,6 +636,28 @@ func TestAppSlugValidation(t *testing.T) {
 	require.ErrorContains(t, err, "duplicate app slug")
 }
 
+func TestMetadataResourceDuplicate(t *testing.T) {
+	t.Parallel()
+
+	// nolint:dogsled
+	_, filename, _, _ := runtime.Caller(0)
+
+	// Load the multiple-apps state file and edit it.
+	dir := filepath.Join(filepath.Dir(filename), "testdata", "resource-metadata-duplicate")
```

Comment:
Suggestion: There's no need to check the `runtime.Caller` here, current working directory is always relative to the test, therefore we can simply do `filepath.Join("testdata, "...")`.

Reply:
Gotcha 👍